docs/tutorials/intro.md

```diff
@@ -168,7 +171,7 @@ and then we can use it in our code:
 use icu_provider_fs::FsDataProvider;
 
 fn main() {
-    let _provider = FsDataProvider::try_new("~/projects/icu/icu4x-data")
+    let _provider = FsDataProvider::try_new("/home/{USER}/projects/icu/icu4x-data")
```

Comment:
Nit: The old one is more general since it works on both Linux and macOS.

Reply:
Unfortunately, the old one doesn't work.